example/storybook/stories/ExampleApp/ExampleApp.stories.tsx

```diff
@@ -32,6 +35,34 @@ storiesOf('Example App', module)
       </DeveloperConfigProvider>
     );
   })
+  .add('With additional navigation tab', () => {
+    const rgbColorString = color('Primary Color', '#0477BF');
+    const primaryColor = Color(rgbColorString).hex();
+
+    return (
+      <DeveloperConfigProvider
+        developerConfig={{
+          simpleTheme: {
+            primaryColor,
+          },
+          additionalNavigationTabs: [
+            {
+              name: 'AdditionalTab',
+              component: HelloWorldScreen,
+              options: {
+                tabBarLabel: t('tabs-settings', 'Settings'),
+                tabBarIcon: Menu,
+              },
+            },
+          ],
+        }}
+      >
+        <RootProviders authConfig={authConfig}>
+          <RootStack />
+        </RootProviders>
+      </DeveloperConfigProvider>
+    );
+  })
```

Comment:
I also made a new story of the full Example app showing an additional navigation tab.  This is moved from the `AppDemo.tsx` way of running the app (without storybook) which doesn't have the ability to show different configurations.

^ @cmjarreau